tests/test_opaque_types.py

```diff
@@ -28,9 +28,10 @@ def test_pointers(msg):
                                 print_opaque_list, return_null_str, get_null_str_value,
                                 return_unique_ptr, ConstructorStats)
 
+    living_before = ConstructorStats.get(ExampleMandA).alive()
```

Comment:
It isn't entirely obvious why this change, but I was getting false positives of this test triggered by another test failure (the other failure resulting in an `ExampleMandA` instance not being destroyed).  This just prevents the false positive when a failure is happening elsewhere.